### PR TITLE
docs: Improve external etcd documentation wording

### LINKED ISSUE #9401: Improve external etcd documentation wording
The external etcd documentation is below:

http://docs.cilium.io/en/latest/gettingstarted/k8s-install-external-etcd/

* The requirements do not list having an external etcd available.
* The section `Configure the External Etcd` is actually for configuring Cilium, not etcd.

We should tidy this up to make it easier to understand for users.

### PR DESCRIPTION
**Pull request by mpeiffer**
Changes to be committed:
	modified:   k8s-install-external-etcd.rst
	modified:   requirements_intro.rst

Previously, having an available external etcd was not listed under the requirements section in the external etcd documentation. This was addressed by listing "Key-Value store etcd >= 3.1.0 or consul >= 0.6.4" under the requirements section. In addition, the section title "Configure the external etcd" referred to the configuration of Cilium and not an external etcd, thus the title was changed to "Configure Cilium" for clarity.

Fixes: #9401

Signed-off-by: Melissa Peiffer mbp83@nau.edu

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/9415)

### CODE REVIEW

File: Documentation/gettingstarted/requirements_intro.rst [hunk 1]
```diff
@@ -6,6 +6,7 @@ Make sure your Kubernetes environment is meeting the requirements:
 * Kubernetes >= 1.9
 * Linux kernel >= 4.9
 * Kubernetes in CNI mode
+* Key-Value store etcd >= 3.1.0 or consul >= 0.6.4
```
Comment (joestringer) [inline Documentation/gettingstarted/requirements_intro.rst:10]: While this improves the docs for the "External etcd" guide, this also now reports such a requirement for the "etcd operator" guide (http://docs.cilium.io/en/latest/gettingstarted/k8s-install-etcd-operator/) which actually requires neither `etcd` nor `consul` to be installed, as that guide will configure `etcd` as part of the guide.

Also, this text is only referred to from etcd guides so the Consul mention is not necessary. That option is already documented here: http://docs.cilium.io/en/latest/install/system_requirements/#summary

```
$ git grep requirements_intro Documentation/
Documentation/gettingstarted/k8s-install-etcd-operator.rst:.. include:: requirements_intro.rst
Documentation/gettingstarted/k8s-install-external-etcd.rst:.. include:: requirements_intro.rst
```

**Review by joestringer — CHANGES_REQUESTED** [review]
Hi, thanks for the contribution!

As a matter of policy, all commits should be signed off (`git commit -s`): http://docs.cilium.io/en/latest/contributing/contributing/#developer-s-certificate-of-origin

I also commented on the changes below, minor adjustmentments needed.

**Comment by joestringer** [comment]
test-docs-please

**Comment by aanm** [comment]
test-docs-please

**Review by joestringer — COMMENTED** [review]
Looks good, could you just rebase and squash the commits together?

https://thoughtbot.com/blog/git-interactive-rebase-squash-amend-rewriting-history

**Comment by aanm** [comment]
ping @mpeiffer

**Comment by mpeiffer** [comment]
Hello, I apologize for the wait. I will rebase and squash my commits together later today.

**Comment by mpeiffer** [comment]
I'm not really sure what I did but I will revert these changes and fix all of this as soon as possible.

**Pull request closed without merging**